gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/storage/StorageDataWrapper.java

```diff
@@ -0,0 +1,14 @@
+package uk.nhs.adaptors.pss.translator.storage;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class StorageDataWrapper {
+    private String type;
+    private String conversationId;
+    private byte[] data;
+}
```

Comment:
Do we also need to store the filename as it will be used in the FHIR `Attachment.title` property.

Reply:
Not a problem at all, I've readjusted the model to include filename.

Reply:
Actually on second thought, I've created a new placeholder model for the storageDownloadWrapper. It's not required on the upload but is required on the download which will be designed in a later ticket.